src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -67,7 +68,7 @@ void HRVOAgent::computeNeighbors()
     Vector current_dest = path.getCurrentPathPoint().value().getPosition();
 
     float new_neighbor_dist =
-        std::min(static_cast<double>(neighborDist_),
+        std::min(static_cast<double>(neighbor_dist_),
```

Comment:
GitHub doesn't allow me to add a comment on line 88, but the variable `openingAngle` should also be updated.